src/test/java/com/patrickjones/services/TransferManagerTest.java

```diff
@@ -0,0 +1,32 @@
+package com.patrickjones.services;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.patrickjones.models.BancAccount;
+
+public class TransferManagerTest {
```

Comment:
This is a unit test class. Notice that it is the name of the class being tested (`TransferManager`) + `Test`. These files have to either begin or end with the word `Test` so that Maven knows to run them when it runs the unit tests.